Service/ContentTypeService.php

```diff
@@ -409,4 +417,57 @@ public function getAllTypes()
         $this->loadEnvironment();
         return implode(',', array_keys($this->contentTypeArrayByName));
     }
+    
+    public function readJson(File $jsonFile, Environment $environment): ContentType
```

Comment:
I think a good name could be "initFromJson", as we are not only reading the json, but also setting the environment.
We are initializing a contentType based on a json.